internal/auth/controller/auth_controller.go

```diff
@@ -95,50 +102,37 @@ func (ac *authController) SignUp(c echo.Context) error {
 	}
 
 	// jwt処理
-	return ac.ah.JwtProcessing(c, resDogOwner)
+	return ac.ah.JwtProcessing(c, resDogOwner, SIGNUP_MSG)
 }
 
-// func (ac *authController) LogIn(c echo.Context) error {
-// 	logger := log.GetLogger(c).Sugar()
-// 	var reqADOD dto.ReqAuthDogOwnerDto = dto.ReqAuthDogOwnerDto{}
-
-// 	if err := c.Bind(&reqADOD); err != nil {
-// 		logger.Error(err)
-// 		return c.JSON(http.StatusBadRequest, wrErrors.ErrorResponse{
-// 			Code:    http.StatusBadRequest,
-// 			Message: "Invalid format",
-// 		})
-// 	}
-// 	logger.Infof("Request AuthDogOwner info: %v", reqADOD)
+// LogIn: login機能
+//
+// args:
+//   - echo.Context: c Echoのコンテキスト。リクエストやレスポンスにアクセスするために使用されます。
+//
+// return:
+//   - error: error情報
+func (ac *authController) LogIn(c echo.Context) error {
+	logger := log.GetLogger(c).Sugar()
 
-// 	// LogIn処理
-// 	resAuthDogOwner, err := ac.ah.LogIn(c, reqADOD)
+	reqADOD := dto.ReqAuthDogOwnerDto{}
 
-// 	if err != nil {
-// 		return c.JSON(http.StatusBadRequest, wrErrors.ErrorResponse{
-// 			Code:    http.StatusBadRequest,
-// 			Message: "Invalid Request",
-// 		})
-// 	}
+	if err := c.Bind(&reqADOD); err != nil {
+		wrErr := errors.NewWRError(err, "入力項目に不正があります。", errors.NewDogownerClientErrorEType())
+		logger.Error(wrErr)
+		return wrErr
+	}
 
-// 	// 秘密鍵取得
-// 	secretKey := configs.FetchCondigStr("os.secret.key")
+	// LogIn処理
+	resDogOwner, wrErr := ac.ah.LogIn(c, reqADOD)
 
-// 	// jwt token生成
-// 	signedToken, err := createToken(secretKey, resAuthDogOwner.DogOwnerID, 72)
-// 	if err != nil {
-// 		return c.JSON(http.StatusInternalServerError, wrErrors.ErrorResponse{
-// 			Code:    http.StatusInternalServerError,
-// 			Message: "Failed to sign token",
-// 		})
-// 	}
+	if wrErr != nil {
+		return wrErr
+	}
 
-// 	return c.JSON(http.StatusCreated, success.SuccessResponse{
-// 		Code:    http.StatusOK,
-// 		Message: "Successful login",
-// 		Token:   signedToken,
-// 	})
-// }
+	// jwt処理
+	return ac.ah.JwtProcessing(c, resDogOwner, LOGIN_MSG)
```

Comment:
loginのエンドポイント（jwt発行)にログインできましたのメッセージはいらないかな。
発行したjwtだけでいいと思うよ。
全般、RestApiで、成功メッセージはレスポンスにはいらないですね。

あと、できれば、c.Jsonのreturnはcontrollerでしたいから、
JwtProcessing()でjwt stringだけ受け取って、それをcontroller側で、jsonにして返してあげた方がいいと思います。
```suggestion
        token := ac.ah.JwtProcessing(c, resDogOwner)
	return c.JSON(http.StatusCreated, map[string]string{
		"accessToken": token,
	})
```